components/colorpicker/overview.md

```diff
@@ -10,92 +10,100 @@ position: 0
 
 # Blazor ColorPicker Overview
 
-The <a href = "https://www.telerik.com/blazor-ui/colorpicker" target="_blank">ColorPicker for Blazor</a> is an interactive component that allows color selection from a popup palette or a [HSVA](https://en.wikipedia.org/wiki/HSL_and_HSV) canvas. Users can also type a specific RGB/HEX color value manually. The ColorPicker is practically identical to the [FlatColorPicker component]({%slug flatcolorpicker-overview%}) with the only difference that the ColorPicker takes up less space and displays the color selection UI in a popup.
+The <a href="https://www.telerik.com/blazor-ui/colorpicker" target="_blank">Telerik UI ColorPicker for Blazor</a> is an interactive component that allows users to select colors from a popup palette or a [Hue, Saturation, Value (HSV) canvas](https://en.wikipedia.org/wiki/HSL_and_HSV), and also manually type a specific RGB or HEX color value.
 
-#### In this article:
-   * [Basics](#basics)
-   * [Example](#example)
-   * [Interface](#interface)
-   * [Features](#features)
-   * [Supported Value Formats](#supported-value-formats)
+Practically, the ColorPicker is identical to the [Telerik UI for Blazor FlatColorPicker component]({%slug flatcolorpicker-overview%}) with the only difference that the ColorPicker takes up less space and displays the color selection UI in a popup.
 
-## Basics
+## Creating the Blazor ColorPicker
```

Comment:
Articles can be omitted in titles and headings. Also, the component is not one specific instance, so "the" sounds a bit strange.

```suggestion
## Creating Blazor ColorPicker
```